packages/react-native/Libraries/Text/RCTTextAttributes.mm

```diff
@@ -20,11 +20,7 @@ @implementation RCTTextAttributes
 // [macOS
 + (RCTUIColor *)defaultForegroundColor
 {
-  if (@available(iOS 13.0, *)) {
-    return [RCTUIColor labelColor];
-  } else {
-    return [RCTUIColor blackColor];
-  }
+  return [RCTUIColor labelColor];
```

Comment:
Does this mean iOS 12 and lower are no longer supported?

Reply:
Nope, this was just an old check I found that I realized we could get rid of while we're at it.